packages/deposit/pubspec.lock

```diff
@@ -1,12 +1,341 @@
 # Generated by pub
 # See https://dart.dev/tools/pub/glossary#lockfile
 packages:
+  _fe_analyzer_shared:
```

Comment:
Should all these changes be added to the lock file? You only added the test dependency.

Reply:
Seems they get added as child dependencies from test